### ISSUE TITLE
how to bind 'pm:create' event on two separate custom controls whose shapes are both 'Polyline'

### ISSUE THREAD

**Issue by Amber711** [geoman-io/leaflet-geoman#641]
Hi all, 

in my case, I have to create two Polyline controls for two separate things(one is to draw navigation route, another to draw something like virtual walls). 
here is the code for creating the two controls:
`mapEle.pm.Toolbar.copyDrawControl('Polyline', {
        name: 'PolylineCopy1',
        title: 'Draw a Navigation Route',
        onClick: () => {
          mapEle.pm.setPathOptions({
            color: '#8dc6ff',
            fillColor: 'green',
            fillOpacity: 0.4,
          });
        },
      });

      mapEle.pm.Toolbar.copyDrawControl('Polyline', {
        name: 'PolylineCopy2',
        title: 'Draw a Virtual Wall',
        className: 'draw-wall-btn',
        onClick: () => {
          mapEle.pm.setPathOptions({
            color: 'green',
            fillColor: 'green',
            fillOpacity: 0.4,
          });
        },
      });`
Now I bind the 'pm:create' event to the `map(mapEle.on('pm:create', e => {
        console.log('new: ', e);
      });),` 

The problem here is, drawing navigation route or virtual wall  calls the same event handler, and it's hard to tell what is it  one is trying to draw....

Has anyone has seen this issue before and how to solve it please?

Thank you!

**Comment by Falke-Design** [comment]
@Amber711 thx for reporting! I will fix this.

Workaround for you:
```
map.pm.Toolbar.copyDrawControl('Polyline', {
  name: 'PolylineCopy1',
  title: 'Draw a Virtual Wall',
  className: 'draw-wall-btn',
});
// set the pathoptions for the new shape and not in the onClick. Else every shape changes the color.
map.pm.Draw.PolylineCopy1.setPathOptions({
  color: '#8dc6ff',
  fillColor: 'green',
  fillOpacity: 0.4,
  layer: 'Copy1' // define a custom option with a name
});
map.pm.Toolbar.copyDrawControl('Polyline', {
  name: 'PolylineCopy2',
  title: 'Draw a Virtual Wall',
  className: 'draw-wall-btn',
});
map.pm.Draw.PolylineCopy2.setPathOptions({
  color: 'green',
  fillColor: 'green',
  fillOpacity: 0.4,
  layer: 'Copy2' // define a custom option with a name
});
map.on('pm:create',e => {console.log(e.layer.options.layer)}) // readout the custom option with the name
```

**Comment by Amber711** [comment]
Hi @Falke-Design ,

Thank you so much for the workaround. Have tried it and that works well!

**Comment by Falke-Design** [comment]
Fixed in Release:  `2.7.0`
